src/ionic.js

```diff
@@ -0,0 +1,10 @@
+import '@ionic/core/css/ionic.css'
+import 'ionicons/dist/collection/icon/icon.css'
+import '@ionic/core/dist/ionic/svg'
+import { defineCustomElements } from '@ionic/core/dist/esm'
+
+export default {
+  init(opts = {}) {
+    defineCustomElements(window, opts)
```

Comment:
what if this is SSR? then `window` doesn't exist

Reply:
That is a great and very valid point and I believe we should bring this up with Ionic as this is how they're initializing as well AFAIK:
https://github.com/ionic-team/ionic/blob/master/angular/src/app-initialize.ts#L9